<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">    QWidget {
        background-color: &quot;green&quot;;
        color: &quot;white&quot;;
    }</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>721</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>451</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Файлы будут скопированы из папки:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>371</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Выберите флешку (куда копировать картинки):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>661</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>661</width>
       <height>61</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>New Item</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>121</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Копировать</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>451</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Файлы будут скопированы из папки:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>661</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>121</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Копировать</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>121</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Повторить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>641</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Не обнаружено подключенных USB устройств. &lt;/p&gt;&lt;p&gt;Попробуйте вынуть USB устройство и вставить обратно. &lt;/p&gt;&lt;p&gt;После этого нажмите кнопку &amp;quot;Повторить&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>500</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Копирование файлов:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>500</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Скопировано 2548 файлов в папку &amp;quot;На печать 10.05.2023&amp;quot;.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Флешка отключена. Ее можно безопасно вынуть.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
